src/feat/pitch-functions.cc

```diff
@@ -1076,7 +1096,17 @@ void OnlinePitchFeatureImpl::AcceptWaveform(
 
   for (int32 frame = start_frame; frame < end_frame; frame++) {
     // start_sample is index into the whole wave, not just this part.
-    int64 start_sample = static_cast<int64>(frame) * frame_shift;
+    int64 start_sample;
+    if (opts_.snip_edges) {
+      // Usual case offset starts at 0
```

Comment:
comments should be properly punctuated and capitalized sentences (not always true in old code, but we're trying to do it going forward).